Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -480,14 +481,19 @@ protected override void GetChildItems(string path, bool recurse)
                             {
                                 GetChildItems(bucket.Name, true);
                             }
-                            catch (Exception e)
+                            catch (GoogleApiException e) when (e.HttpStatusCode == HttpStatusCode.Forbidden) { }
```

Comment:
Added a comment about why this might occur and that ignoring those objects is intended.